textproc/pandoc/Portfile

```diff
@@ -24,33 +25,45 @@ long_description    \
     Textile, groff man pages, plain text, Emacs Org-Mode, AsciiDoc, EPUB (v2 \
     and v3), FictionBook2, and S5, Slidy and Slideous HTML slide shows.
 
-depends_lib-append  \
-                    port:hs-aeson \
-                    port:hs-alex \
-                    port:hs-attoparsec \
-                    port:hs-base64-bytestring \
-                    port:hs-blaze-html \
-                    port:hs-blaze-markup \
-                    port:hs-data-default \
-                    port:hs-extensible-exceptions \
-                    port:hs-happy \
-                    port:hs-highlighting-kate \
-                    port:hs-hslua \
-                    port:hs-http \
-                    port:hs-mtl \
-                    port:hs-network \
-                    port:hs-pandoc-types \
-                    port:hs-parsec \
-                    port:hs-random \
-                    port:hs-syb \
-                    port:hs-sha \
-                    port:hs-tagsoup \
-                    port:hs-temporary \
-                    port:hs-texmath \
-                    port:hs-text \
-                    port:hs-unordered-containers \
-                    port:hs-vector \
-                    port:hs-xml \
-                    port:hs-yaml \
-                    port:hs-zip-archive \
-                    port:hs-zlib
+depends_build-append  \
+                    port:stack
+
+set system_gcc      /usr/bin/gcc
+
+use_configure       no
+
+# the PATH environment must provide /usr/bin/gcc, not ${prefix}/bin/gcc
+build.env \
+    PATH=/usr/bin:$env(PATH) \
+    PREFIX=${prefix} \
+    CC=${system_gcc}
+
+build {
+    xinstall -m 0755 -d ${workpath}/.stack
+    # the PATH environment must use /usr/bin/gcc, not ${prefix}/bin/gcc
+    system -W ${worksrcpath} \
```

Comment:
This looks as if it could be split into configure, build, destroot phase? Not that it is wrong this way.

Reply:
I’ll go with consensus. I think of `configure` as the step that makes the Makefile, which isn’t being done by `setup`, but it is analogous.

Reply:
I agree with @mojca here. Configure phase is not just for making the make file, by any checks and configurations that are performed before the actual build starts. The first step here, setup, looks like it should be done in this phase. Likewise the final install step should be moved to the destroot phase.

Reply:
> I had to hack a nonexistent `configure.target` with `configure.args`. Is there a better way?

If it doesn't exist, it might be a valid request to ask for official support of `configure.target` in the future. For now you probably just need to place it inside one of the 'arguments' of `configure` (`configure.cmd`, `configure.pre-args`, `configure.args`, ... whatever makes more sense), so I guess that it's ok what you have now.